content/articles/cloudflare-service.markdown

```diff
@@ -16,10 +24,45 @@ CloudFlare has two integration methods:
+
+### DNSimple one-click integration
+
+DNSimple one-click integration allows you to use CloudFlare to proxy only a specific DNS record of a domain. This is the preferred option, for example, if you want to take advantage of CloudFlare's CDN and protection for the main website, maintaining the other DNS records in DNSimple.
+
+However, this approach has some limitations. For instance, CloudFlare doesn't allow to use the Universal SSL feature for domains or subdomains managed from a third-party provider. More information below.
+
+
 ## DNS limitations
 
-CloudFlare caches your site content at locations across the globe. It uses DNS to route your requests to one of these locations depending on various factors. To support this technology within DNS limitations it is best to use a host name (i.e. www.example.com) rather than your domain apex (i.e. example.com). CNAME records at a domain apex are disallowed or will cause your DNS respond incorrectly. You may wish to use DNSimple's URL redirection record to forward requests from your domain apex to "www" for instance.
+CloudFlare caches your site content at locations across the globe. It uses DNS to route your requests to one of these locations depending on various factors. To support this technology within DNS limitations it is best to use a host name (i.e. www.example.com) rather than your domain apex (i.e. example.com). CNAME records at a domain apex are disallowed or will cause your DNS respond incorrectly. You may wish to use DNSimple's URL redirection record to forward requests from your domain apex to "www" for instance.
+
 
 ## ALIAS record limitations
 
 DNSimple offers a virtual record type called an ALIAS record. This allows sites to work around a limitation of CNAME records at the domain apex by resolving to an A record at the name server. Because CloudFlare may use the name server physical location to determine how your request will be routed it would short circuit the utility of CloudFlare. As such we advise against using ALIAS records with CloudFlare.
+
+
+## Universal SSL limitations
+
+[CloudFlare Universal SSL](https://www.cloudflare.com/ssl) is currently available only if you use Cloudflare directly (first integration method). It's not possible to use CloudFlare Universal SSL if you enable Cloudflare using DNSimple one-click service.
+
+
+## Enabling CloudFlare service {#enable}
+
+From the domain page, switch to the <label>Services</label> tab and click to <label>Add or edit service</label>. Find the **CloudFlare** service, click to <label>Add</label> and follow the instructions
+
+
+## Disabling ClouldFlare service {#disable}
+
+<warning>
+A known bug currently prevents the possibility to disable the CloudFlare service from the UI. Please [contact us](https://dnsimple.com/contact) if you want to disable the service.
+</warning>
```

Comment:
Should we include a note about how to workaround this issue? It's possible to workaround, but it's rather cumbersome.